js/app.js

```diff
@@ -0,0 +1,50 @@
+$(function () {
+
+  // Use ```$.click``` to trigger the application to conduct 
+  // a city search when the user clicks the "submit" button
+  $("#submit-btn").click(citySearch);
+
+    function citySearch() {
+
+        // Call ```$.val()``` on inputs to get the string value of your user's input
+        // Store user input in a variable called ```city```
+        var city = $("#city-type").val(); 
```

Comment:
@kvelez1 consider converting the input to the same case as you condition, as that would consider come input validations.